developer/applications/vehicle-collision/index.html

```diff
@@ -20,10 +20,6 @@ <h3>Tools for visualizing and analyzing vehicle collisions</h3>
         class="fab fa-github"></i><span class="sr-only">github repository link</span> Repository</a>
     <a class="application__repo" href="https://github.com/agrc/crash-db"><i aria-hidden="true"
         class="fab fa-github"></i><span class="sr-only">github repository link</span> Repository</a>
-    <a id="application__download" class="application__download"
```

Comment:
this is kind of important. why did you remove it? Does it not work?

Reply:
I was thinking "roadkill" since that's what the link to this page in #1565 showed. I just figured that it was a copy/paste error. I'll put it back.

Reply:
ok, i updated the issue since this isn't roadkill. I don't think we have a spotlight for it in the developer applications section yet